### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by bellom**
Kindly review

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (nidalaa) [inline package.json:1]: Make sure you also use `package.lock` file. Thanks to this you will avoid problems with future versions of the dependencies 💪

**Review by nidalaa — CHANGES_REQUESTED** [review]
Hi @bellom ! Nice job here :)

This is not a full review - as we agreed I am here testing some of the GH features :)

I lef one comment though :)

**Review by dipto0321 — APPROVED** [review]
Hi @bellom ,

Good job 👏 . This is a basic app for testing your learning on webapck. So I didn't find any major issues. Except for this one 

After fixing this design you can merge it. This is from me 👍 

[**Note:** Here is a good [tutorial ](https://www.youtube.com/watch?v=MpGLUVbqoYQ) about webpack. You can go for it to gain more about webpack]

Thanks 🙏

**Comment by bellom** [comment]
Design didn't break on my system.

**Comment by dipto0321** [comment]
HI @bellom ,

Then Go ahead. 🙂 

Thanks 🙏

**Pull request merged**